lib/solidus_support/engine_extensions.rb

```diff
@@ -46,14 +49,42 @@ def load_solidus_subscribers_from(path)
         end
       end
 
-      # Loads decorator files.
+      # Loads decorators.
```

Comment:
We've started the move to using "overrides" instead of "decorators" (because that's what the Rails docs call them and also this isn't the decorator pattern.) Do we want to take this continue moving that direction?

Reply:
For reference this has been discussed before in https://github.com/solidusio/solidus/issues/3010

Rephrasing my previous comment from above mentioned issue on "overrides" here:

> But, I wholeheartedly disagree with calling them `*_overrides.rb`. These things are monkey patches. Please, let's call them what they are then (`*_monkey_patch.rb`s). Everybody knows what a monkey patch is. We even call them monkey patches every time we explain what these "decorators" are meant for. So, why another disguise?
> 
> Also, Deface, a still widely used gem in Solidus ecosystem, already uses the `app/overrides` folder*.

_*) https://github.com/solidusio/solidus/issues/4877 is still an issue because of the wide spread usage of Deface in our eco system._

Not sure I would still call them "monkey patches" nowadays (it's kind of "toy language" and sounds like something has been desperately made out of emergency reasons). Since this pattern is so common in our eco system I would like to give it a better, more formal name. Ruby itself coined the term "Refinements", but we cannot use them for obvious reasons. 

So, what are we left with? "Extensions", "Prependers" (as mentioned in https://github.com/solidusio/solidus_support/pull/60), "Overloads"?

Reply:
`overrides` is problematic, because the Deface gem uses `load` to load things in `app/overrides`, regardless of whether it's a deface DSL file or any other Ruby file. I can imagine using `app/patches` instead: That would be less derisive as a name, but still indicate patching core classes is not entirely desired.

Reply:
I really want to follow Rails conventions on this, but understand the impasse. I propose we hold off on changing anything as part of this PR and come up with a good solution later. (Sorry for bringing it up! I typically just use the Deface DSL and hadn't run into these issues!)

Reply:
Let's make a Rails convention then :) https://github.com/rails/rails/pull/53714

Reply:
The Rails team doesn't like patches, and would prefer not to encourage people. I'll try getting something together that implements more or less what's in that PR in a separate gem, usable for the Solidus ecosystem, the Alchemy ecosystem, and, importantly, for host apps. 

While I understand the arguments about patches being a bad practice, I also understand they're an extremely powerful tool that we all often reach for, and should therefore support in a way that works well.

Reply:
Thanks @mamhoff. That sounds great to me.